src/styles/main.css

```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+}
```

Comment:
Don't reset margins using * selector
Better to specify the elements, for which you want to reset margins